mozilla/foundation.mozilla.org
# Wagtail data

Related issue: #1308
Summary: Updates the load_fake_data script to generate Wagtail pages and other content. Also adds a post deploy step for review apps that adds a super user and logs the password to the build logs.

- [x] Build pages
- [x] Remove crufty generate steps
- [x] update build steps 
- [x] Build homepage highlights


To test:

1. Run from a freshly migrated DB: `inv manage load_fake_data`
2. Run a second time: `inv manage load_fake_data`
3. Run with the delete Flag: `inv manage load_fake_data -f delete`

After each step, check the site, and the CMS for issues (pages load, no duplicate homepages/sites, etc)

Also, you can run the code that creates a super user with `inv manage review_app_admin`
